plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+project(obs-webrtc)
+
+find_package(Corrosion QUIET)
+if(NOT Corrosion_FOUND)
+  include(FetchContent)
+  FetchContent_Declare(
+    Corrosion
+    GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+    GIT_TAG v0.3.1)
+  FetchContent_MakeAvailable(Corrosion)
+endif()
+
+set(OBS_WEBRTC_GENERATED_DIR "${CMAKE_CURRENT_BINARY_DIR}/generated")
+
+corrosion_import_crate(MANIFEST_PATH webrtc/Cargo.toml)
+corrosion_set_env_vars(webrtc-rs-lib
+                       "OBS_WEBRTC_GENERATED_DIR=${OBS_WEBRTC_GENERATED_DIR}")
+
+# Force dependent crates to link against the correct deployment target
+if(OS_MACOS)
```

Comment:
Can this be moved into the macOS branch further below?

Reply:
It can, I was just trying to keep locality for easier understandability